### PR TITLE
CMake translations

### PR DESCRIPTION
**Pull request by redtide**
- Added CMake for installation (will be used for AUR package) and Gettext translation builds
- desktop file built with CMake from a template
- EditorConfig file to respect indentation and other editing rules
- gitignore to avoid unneeded files mixed with others
- translation template file and Italian translation for testing

~~The translation seems not working yet, I need to get some practice with it, it's new to me using Gettext with Bash scripts, but AFAIU from documentation most of the work should be done already~~.

Note that the binary was renamed to .sh to ease the translation process, CMake will install it without the extension and with proper file permissions.

The desktop file was a bit modified, note the removal of the Encoding key, which seems to be a [deprecated key](https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html#deprecated-items)

Feel free to edit the translation as needed. Qt Linguist can translate `.po` files but I suggest to install and use `poedit`, which can load `.pot` templates and create/update `.po` files from them. Hope to see this in Weblate :)

mainwindow
saveas
saveclip

### CODE REVIEW

**Comment by stefonarch** [comment]
Translations and installation working fine, but "selected area" doesn't work anymore, either started from desktop file or cli.

File: wshot.sh [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    else
+    zenity \
+      --title=Wshot \
+      --width=1500 \
+      --warning \
+      --timeout 3 \
+      --text="$_winDetectionError"
+    GEO="$(slurp)"
+    fi
+  elif [ "$mode" == "_$selectedArea" ];then
```
Comment (stefonarch) [inline wshot.sh:91]: "_$selectedArea" → "$_selectedArea" fix the issue.

**Review by stefonarch — CHANGES_REQUESTED** [review]

File: wshot.sh (hunk 1 above)
Comment (redtide) [reply wshot.sh:91]: ouch typo, going to fix it!

**Comment by redtide** [comment]
Should be alright, including the combo to checkbox replacements.

**Review by stefonarch — APPROVED** [review]
Thanks, all fine now.

**Pull request merged**